### DISCUSSION TITLE
Progress from expeditions before save file

### DISCUSSION THREAD

**Discussion by Kelador** [leodium/leodium-ReLive-NoMansSky-Expeditions-Offline#15] [General]
Hi last expedition I played was 12 I have completed all up till that if I download 13 does it have all the rewards unlocked from the past rewards as if you completed or will I loose that progress?

**Comment by sgallaty** [comment]
No sir.  Expeditions are like side-quests to your main save.

These are no different, it's just a way to trigger them at any time.  You
get all rewards from the expeditions just like you did a regular one and
the import/export terminals work, even with expedition specific ships and
modules for the most part.

On Sun, Aug 11, 2024 at 11:24 AM Kelador ***@***.***> wrote:

> Hi last expedition I played was 12 I have completed all up till that if I
> download 13 does it have all the rewards unlocked from the past rewards as
> if you completed or will I loose that progress?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/leodium/leodium-ReLive-NoMansSky-Expeditions-Offline/discussions/15>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAUGOQM6NWCTE3R7STKMDUTZQ6M4VAVCNFSM6AAAAABMK5MBKKVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZXGAZTOOBSHA>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID:
> <leodium/leodium-ReLive-NoMansSky-Expeditions-Offline/repo-discussions/15@
> github.com>
>


-- 
S. Patrick Gallaty
***@***.***

**Reply by cwmonkey** [reply]
Saying "no" is kind of confusing here, since they asked if it did one thing or a different thing 🤔

**Comment by cwmonkey** [comment]
Many rewards are unlocked account-wide, meaning any save will have access to them from the QuickSilver vendor on the Anomaly. When you start a new expedition you will always have a "new" character that can go re-claim the rewards from the QuickSilver vendor. However since expedition 13 has no NPCs at all, there is nobody on the Anomaly to claim your rewards from. Expeditions 3 and 9 are similar in that you cannot get to the Anomaly until the expedition is almost done, so you have to make due with what you are given for the expedition.

I put "new" in quotes because when you start an expedition on the Anomaly with an existing save, your traveler's appearance is copied over to an expedition save with a new multitool, ship, etc. During the expedition you can switch to your main save (to copy your expedition multitool and/or ship for nanites from the expedition terminal). In a very real sense, when you start an expedition from the terminal on the Anomaly, you are playing a save file within your save file.
